Fix missing usages of DESTDIR
--- a/gpMgmt/bin/Makefile
+++ b/gpMgmt/bin/Makefile
@@ -55,7 +55,7 @@
 PYLIB_DIR=$(SRC)/ext

 core: pygresql subprocess32
-	python gpconfig_modules/parse_guc_metadata.py $(prefix)
+	python gpconfig_modules/parse_guc_metadata.py $(DESTDIR)$(prefix)

 ifneq "$(wildcard $(CURDIR)/pythonSrc/ext/*.tar.gz)" ""
 install: installdirs installprograms core lockfile psutil
@@ -79,7 +79,7 @@
 		unset LIBPATH; \
 		./generate-greenplum-path.sh $(prefix) > $(DESTDIR)$(prefix)/greenplum_path.sh ; \
 	fi
-	. $(prefix)/greenplum_path.sh && unset PYTHONHOME && \
+	. $(DESTDIR)$(prefix)/greenplum_path.sh && unset PYTHONHOME && \
 	if [ "$(BLD_ARCH)" = 'aix7_ppc_64' ]; then \
 	    unset PYTHONPATH && cd $(PYLIB_SRC)/$(PYGRESQL_DIR) && DESTDIR="$(DESTDIR)" CC="$(CC)" python_64 setup.py build; \
 	elif [ `uname -s` = 'OpenBSD' ]; then \
--- a/gpcontrib/gpcloud/Makefile
+++ b/gpcontrib/gpcloud/Makefile
@@ -32,7 +32,7 @@
 install: install-symlink

 install-symlink:
-	ln -sf gpcloud.so $(pkglibdir)/gps3ext.so
+	ln -sf gpcloud.so $(DESTDIR)$(pkglibdir)/gps3ext.so

 test: format
 	@$(MAKE) -C test test
